_data/accessibility-tests/identifier.yml

```diff
@@ -0,0 +1,73 @@
+title: Identifier
+component_status: pass
+test_items:
+# General tests
+  - summary: Identifier appears consistently. 
+    summary_additional: The identifier appears in the same location across all pages of the website.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.3
+# Zoom/screen magnification tests
+  - summary: Identifier links are visible and functional when screen is magnified.
+    summary_additional: When you zoom in to 400%, identifier links are stacked and visible. No left-to-right scrolling is required.
```

Comment:
I noticed that for the How to test with zoom magnification section, 200% is used. This has 400%. Was that desired here but only here? @amycole501 or @alex-hull ?

Reply:
For that _reflow_ test yes it should be 400%. 

_Text resize_ is for 200%. We need to go back to some of the previous checks that have been published to make that distinction more clearly. But for the identifier component, we could change "resize text's 1.4.4 prompt to read:
**Identifier text and images of text can be resized up to 200%.**  When you zoom in to 200%, text remains readable and visible within the identifier.

Reply:
Discussed with @cathybaptista. I will go into branch and provide this new edit for the second test under zoom mag.

Reply:
@amycole501 
cc: @finekatie 
I am confused about the addition of "images of text" in the summary here. There are currently no images of text in the identifier (expect possibly the user-added logo?), and I don't want to users to think we are suggesting to add images of text. Is it possible to use an existing summary for this criteria, either "Text can be resized without loss of information." (From the [Typography tests page](https://designsystem.digital.gov/components/typography/accessibility-tests/)) or "Text remains visible when screen is magnified." (From the [Banner tests page](https://designsystem.digital.gov/components/banner/accessibility-tests/))?

Reply:
Going to defer to @amycole501 on this one - Amy C, should it be "images" or "images or text" instead?

Reply:
I feel like we can remove that phrasing. I think it makes sense for it to be something like "Text can be resized without loss of information". I think the if anyone is using their logo, they will see that they need to add alt-text as stated in the non-text content criteria.

Reply:
I agree, I think for the sake of consistency we can remove the reference to logos. However maybe somewhere on the page or guidance we should include a reminder that logos with text, which are prevalent in the identifier, require alt text since that shouldn't be overlooked. It can be added later in the interest of publishing the checks page.